src/components/ChatInput/ChatInput.js

```diff
@@ -16,6 +16,11 @@ const ChatInput = () => {
   const inputRef = useRef(null);
   const messageRef = useRef();
   const [disableButton, setDisableButton] = useState(true);
+  const [members, setMembers] = useState({});
```

Comment:
can we handle this state using zustand to avoid prop drilling?

Reply:
Yes, that's a good idea.